### PR TITLE
 Rename FileUtils.rm_rf to Shards::FileUtils.rm_rf_logging

### PR DESCRIPTION
**Pull request by straight-shoota**
See comment in #209

### CODE REVIEW

File: test/support/factories.cr [hunk 1]
```diff
@@ -44,7 +44,7 @@ module Shards
     def create_git_commit(project, message = "new commit")
       Dir.cd(git_path(project)) do
         run "git add ."
-        run "git commit --allow-empty -m '#{message}'"
+        run "git commit --no-gpg-sign --allow-empty -m '#{message}'"
```
Comment (ysbaddaden) [inline test/support/factories.cr]: Maybe useful, but unrelated to this pull request.

Comment (straight-shoota) [reply test/support/factories.cr]: Well, it was required to run the test suite without being asked about signing any random commits.

**Comment by RX14** [comment]
Why not just specify `Shards::FileUtils` every place we use shard's fileutils. That or rename the module.

**Comment by straight-shoota** [comment]
@RX14 Because you don't need to fully qualify the name inside the `Shards` namespace. I don't think it makes sense to rely on voluntarily overspecifying the name to make it obvious what this is targeting.

Using an entirely different name would be fine, though. Any suggestions? `Shards::Tools`? `Shards::Util`? Or maybe just put these methods directly into `Shards` namespace?

**Comment by RX14** [comment]
`Shards.rm_rf` is fine to me.

And personally, I don't mind voluntary stuff like this for small projects.

**Comment by ysbaddaden** [comment]
Or just drop it and move `#escape(path)` to helpers.

Done in125304d9ddfb2bb84e9c6ad2433de5fcdfb613ca

**Pull request closed without merging**